catalog-info.yaml

```
@@ -0,0 +1,85 @@
+# Declare a Backstage Component for cloud-on-k8s-operator
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: cloud-on-k8s-operator
+  description: Elastic Cloud on Kubernetes (ECK)
+  annotations:
+    github.com/project-slug: elastic/cloud-on-k8s
+    buildkite.com/project-slug: elastic/cloud-on-k8s
+spec:
+  type: service
+  owner: group:cloud-on-k8s
+  system: control-plane
+  lifecycle: production
+
+# Declare our Buildkite pipelines
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: cloud-on-k8s-buildkite-pipeline
+  description: Buildkite Pipeline for cloud-on-k8s-operator
+  links:
+  - title: Pipeline
+    url: https://buildkite.com/elastic/cloud-on-k8s
```

Should this be the link to the actual pipeline?
```suggestion
    url: https://buildkite.com/elastic/cloud-on-k8s-operator
```